JenkinsfileRT_dev

```diff
@@ -95,9 +95,19 @@ bc0.test_configs = [data_config]
 bc1 = utils.copy(bc0)
 bc1.nodetype = 'macos'
 bc1.name = 'macos-unstable-deps'
-bc1.build_cmds = ["pip install -e ."]
+bc0.pip_reqs_files = ['requirements-dev.txt']
```

Comment:
Just noticed this now, after the fact. Should this be `bc1` here, as opposed to `bc0`?  Looks like a copy-n-paste error?

Reply:
good catch; https://github.com/spacetelescope/jwst/pull/7504